exporters/noop/noop_exporter_test.go

```diff
@@ -16,18 +16,18 @@ var ne = nc.New()
 
 func TestExporterByName(t *testing.T) {
 	logger, _ := test.NewNullLogger()
-	exporters.RegisterExporter(noopExporterMetadata.ExpName, nc)
-	ne, err := exporters.ExporterByName(noopExporterMetadata.ExpName, "", logger)
+	exporters.RegisterExporter(Metadata.ExpName, nc)
+	ne, err := exporters.ExporterByName(Metadata.ExpName, "", logger)
 	assert.NoError(t, err)
 	assert.Implements(t, (*exporters.Exporter)(nil), ne)
 }
 
 func TestExporterMetadata(t *testing.T) {
 	meta := ne.Metadata()
 	assert.Equal(t, plugins.PluginType(plugins.Exporter), meta.Type())
-	assert.Equal(t, noopExporterMetadata.ExpName, meta.Name())
-	assert.Equal(t, noopExporterMetadata.ExpDescription, meta.Description())
-	assert.Equal(t, noopExporterMetadata.ExpDeprecated, meta.Deprecated())
+	assert.Equal(t, Metadata.ExpName, meta.Name())
+	assert.Equal(t, Metadata.ExpDescription, meta.Description())
+	assert.Equal(t, Metadata.ExpDeprecated, meta.Deprecated())
```

Comment:
Seems like we could use the same Metadata object in Export/Importer until we need them to diverge.